app/objects/c_plugin.py

```diff
@@ -2,20 +2,34 @@
 import os
 from importlib import import_module
 
+import marshmallow as ma
+
 from app.objects.interfaces.i_object import FirstClassObjectInterface
 from app.utility.base_object import BaseObject
```

Comment:
for imports put the order as:
standard lib

local imports

3rd party

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Ummm, from the pep 8 docs you linked:

>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.